sorl_thumbnail_serializer/fields.py

```diff
@@ -65,6 +66,7 @@ def __init__(self, geometry_string, options={}, *args, **kwargs):
         """  # NOQA
         self.geometry_string = geometry_string
         self.options = options
+        self.uri_prefix = uri_prefix
```

Comment:
Sorry that's a bit unclear now that I read it myself. I just meant I'd like an the docstring above, under the **Args** heading.